Contributing.md

```diff
@@ -30,10 +30,20 @@ This project uses clang-format (stable branch) to check for common style issues.
 
 In most cases, clang-format can and **should** be used to automatically reformat code and solve most formatting issues.
 
-- To run clang-format on all staged files:
+It can be downloaded as part of the LLVM package here: [LLVM Downloads](http://releases.llvm.org/download.html)
+
+- To run clang-format on all staged files use the following command:
```

Comment:
Is it not obvious that what follows is a command? Anyway, please add a comma after "files" if you're going to keep this change.